mlflow/recipes/steps/train.py

```diff
@@ -162,7 +167,10 @@ def _validate_and_apply_step_config(self):
                 error_code=INVALID_PARAMETER_VALUE,
             )
 
-        self.predict_proba = self.step_config.get("predict_proba", False)
+        self.predict_prefix = self.step_config.get("predict_prefix", "predicted_")
```

Comment:
Just to confirm, have other stakeholders approved this prefix conf for the model `predict()` output names? Just making sure since it's not obvious to me that this needs to be configurable.

Reply:
cc @jinzhang21

Reply:
There is a P1 requirement:
```
[P1] I can set a prefix of the output column names to avoid name collision with input columns.
Example: prefix=”my_”, output becomes {“my_label”: “true”, “my_score”: 0.85}
“predicted_” is the default prefix.
```
So added this conf based on that. 

We don't need to document them right away since we have sensible defaults for it. :)

Reply:
Sounds good!